### PR TITLE
Feature/nuc564 image uploader component

### PR DESCRIPTION
**Pull request by ercang**
image
image

### CODE REVIEW

**Comment by yerol** [comment]
This pull request **introduces 6 alerts** when merging 93dfe94912a73d606932d1a8be0a58a82d3035c7 into cc615887452e8ae75637e7cbe80bd146692413d9 - [view on LGTM.com](https://lgtm.com/projects/g/vuexp/vuexp/rev/pr-0352fb7ed8c7c2b23763e045d95c9f50f2f79ecf)

**new alerts:**

* 6 for Property access on null or undefined

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by yerol** [comment]
This pull request **introduces 6 alerts** when merging a5e649cc957a7deedfb35e29477c27b6db005124 into cc615887452e8ae75637e7cbe80bd146692413d9 - [view on LGTM.com](https://lgtm.com/projects/g/vuexp/vuexp/rev/pr-eaceeff846e243329869c4485f89e01af7d69b0a)

**new alerts:**

* 6 for Property access on null or undefined

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by yerol** [comment]
This pull request **introduces 6 alerts** when merging ef050d85edbabc0a71fa5ee71c36a3a594c8cca1 into cc615887452e8ae75637e7cbe80bd146692413d9 - [view on LGTM.com](https://lgtm.com/projects/g/vuexp/vuexp/rev/pr-97983bf4fb4a74b1c85ede6a9465818c1b843771)

**new alerts:**

* 6 for Property access on null or undefined

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by yerol** [comment]
This pull request **introduces 6 alerts** when merging a8156afeeb36db602ee36bc4a11f15ed92afcec2 into cc615887452e8ae75637e7cbe80bd146692413d9 - [view on LGTM.com](https://lgtm.com/projects/g/vuexp/vuexp/rev/pr-50cfd82819882b62e69727fa794811879ead9276)

**new alerts:**

* 6 for Property access on null or undefined

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by yerol** [comment]
This pull request **introduces 6 alerts** when merging 70ec98260a23e90f38c4411d7de620d1d979b7cf into f1c399d9e20e432f550180b6034a8c6b80e66e22 - [view on LGTM.com](https://lgtm.com/projects/g/vuexp/vuexp/rev/pr-59ffe78f3c3c09012f12a77ce8f35cadea719eb4)

**new alerts:**

* 6 for Property access on null or undefined

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: docs/.vuepress/components/VxpImageUploaderLiveDemo.vue [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+          id="vxpimageuploader_maxfilesize_input"
+          v-model="uploaderMaxSize"
+          class="option-input"
+        />
+        <VxpLabel text="(in bytes)"/>
+      </FlexboxLayout>
+
+      <FlexboxLayout flexWrap="wrap">
+        <VxpLabel text="Disabled" class="option-label"/>
+        <input id="vxpimageuploader_disabled_cb" type="checkbox" v-model="uploaderDisabled">
```
[the remark this answers is not in the document]
Comment (ercang) [reply docs/.vuepress/components/VxpImageUploaderLiveDemo.vue]: updated

File: src/components/VxpImageUploader.vue [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  methods: {
+    onImageTap() {
+      if (this.disabled) {
+        return;
+      }
+
+      this.filePicker.open();
+    },
+    onProgress(percent) {
+      console.log('file upload progress - %' + percent); //eslint-disable-line
```
[the remark this answers is not in the document]
Comment (ercang) [reply src/components/VxpImageUploader.vue]: exposed on-progress event

**Comment by yerol** [comment]
This pull request **introduces 6 alerts** when merging 34ed1c610fcdfee035d8ad6de2db6475dfd338fd into f1c399d9e20e432f550180b6034a8c6b80e66e22 - [view on LGTM.com](https://lgtm.com/projects/g/vuexp/vuexp/rev/pr-9260ed98df4affbd5a59396fec09ebe1844b4d1a)

**new alerts:**

* 6 for Property access on null or undefined

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by yerol** [comment]
This pull request **introduces 6 alerts** when merging 206a15519b9ba87590a6ead70f0f201c599dc937 into 9efece4b0a6d60cea76078f663f8c5d5bb832337 - [view on LGTM.com](https://lgtm.com/projects/g/vuexp/vuexp/rev/pr-b6e5aa85877d16b0f667137098449b95b0aa73b4)

**new alerts:**

* 6 for Property access on null or undefined

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**